### PR TITLE
Login Page: Initialization

### PR DESCRIPTION
**Pull request by LiorKGOW**
Initializing the Login Page
- routing the url to the appropriate function

### CODE REVIEW

**Review by ShellyGolden — COMMENTED** [review]
don't forget to: 
add the part of checking if the ID is already exist/not. 
create a new ID if it's not exist 
:)

**Comment by ronyyosef** [comment]
Update here when it's ready for review

File: house/views.py [hunk 1]
```diff
@@ -11,7 +17,38 @@ def global_page(request):
...
+
+            return HttpResponseRedirect('/../{house_id}')
+        # ////// OR //////
+        # if House.objects.get(house_id=house_id):
+        #    return HttpResponseRedirect('/../{house_id}')
+
+        else:
+            # Create a new House with the given house_id:
+
+            House.objects.create(house_id=house_id)
```
Comment (shlior7) [inline house/views.py:44]: You should create the object using the function and can fill up the data needed from
form.data[]
 You could alsoo, I think there is a way to make the form create the house with its magic we need to check that Django sorcery.

**Comment by LiorKGOW** [comment]
Closing this PR, and opening a different one instead #80

**Pull request closed without merging**